crates/contracts/src/utils/utils.cairo

```diff
@@ -6,6 +6,35 @@ use core::serde::Serde;
 
 use dojo::model::{Layout, FieldLayout};
 
+#[inline(always)]
```

Comment:
We should add some simple cairo tests for those functions. To have a first coverage on the function behavior, and then the one you already did at the contract level.

Reply:
I've splitted the `utils.cairo` file and I've added some tests for every util functions 🫡